config.json

```diff
@@ -1312,7 +1318,12 @@
         "name": "Variable Length Quantity",
         "uuid": "6be39d1f-f68d-4a52-bd72-85a38ffd509e",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": [
+          "array-transformations",
+          "arrow-functions",
+          "functions",
+          "rest-and-spread"
```

Comment:
* `errors` is missing
* most solutions I saw use `for-loops`
* I would recommend to add `arrays` as well because that is where we teach push etc which is often used here
* I don't see the reason why `functions` is mentioned here
* I would recommend to add `bitwise-operators` even though we don't have the concept yet but they seem to be often used here. (All the bugs with exercises not unlocking because of something like this have been fixed now.)